tc/autotuner/options_cache-inl.h

```diff
@@ -317,9 +318,41 @@ void OptionsCache<Backend>::fromProtobuf(
   }
 }
 
-} // namespace autotune
+template <typename Backend>
+std::vector<typename Backend::MappingOptionsType> loadTopKFromCache(
+    const std::string& tc,
+    const std::string& entryPoint,
+    const std::string& cacheFileName,
+    const std::vector<const DLConstTensor*>& inputs,
+    const size_t numCandidates) {
+  LOG(INFO) << "Loading proto from: " << cacheFileName << std::endl;
+  if (cacheFileName.empty()) {
```

Comment:
Now that this is a separate function, let's get rid of this special-casing from the tuner that was necessary to avoid saving.  Just let it go further and throw on attempting to open a non-existent file.